pylint/config/man_help_formatter.py

```diff
@@ -4,10 +4,10 @@
 import optparse  # pylint: disable=deprecated-module
 import sys
 import time
+from abc import ABCMeta
 
 
-# pylint: disable=abstract-method; by design?
-class _ManHelpFormatter(optparse.HelpFormatter):
+class _ManHelpFormatter(optparse.HelpFormatter, metaclass=ABCMeta):
```

Comment:
This shouldn't really be in a `typo` PR.. Perhaps do a quick separate PR?

Reply:
I got carried away, sorry :smile:

Reply:
Are you going to make a separate PR for this? Or should I add a suggestion to revert it so you can apply it?

Reply:
I'm going to rebase and separate that.